Datadog/Example/Scenarios/RUM/StopSessionScenario/KioskSendEventsViewController.swift

```diff
@@ -0,0 +1,60 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-Present Datadog, Inc.
+
+import UIKit
+
+internal class KioskSendEventsViewController: UIViewController {
+    @IBOutlet private var doneButton: UIButton!
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        doneButton.isHidden = true
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        rumMonitor.startView(viewController: self, name: "KioskSendEvents")
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+
+        rumMonitor.stopView(viewController: self)
+    }
+
+    @IBAction func didTapDownloadResourceButton(_ sender: Any) {
```

Comment:
If I'm not mistaken you can change the sender type to `UIButton` and avoid force cast